examples/scheduler-policy-config-with-extender.json

```diff
@@ -14,15 +14,15 @@
 	{"name" : "ServiceSpreadingPriority", "weight" : 1},
 	{"name" : "EqualPriority", "weight" : 1}
 	],
-"extenders":[ 
+"extender": 
```

Comment:
Is there a design doc that explains the rational for the loss of multiple extenders :) ?